sources/libClangSharp/ClangSharp.h

```diff
@@ -722,6 +729,8 @@ CLANGSHARP_LINKAGE CXCursor clangsharp_Cursor_getUsedContext(CXCursor C);
 
 CLANGSHARP_LINKAGE CXCursor clangsharp_Cursor_getVBase(CXCursor C, unsigned i);
 
+CLANGSHARP_LINKAGE int64_t clangsharp_Cursor_getDtorVtblIdx(CXCursor C, CX_DestructorType dtor);
```

Comment:
The corresponding entry needs to be added to just before https://github.com/microsoft/ClangSharp/blob/main/sources/ClangSharp.Interop/clangsharp.cs#L870 for the C# side, same with adding the enum definition for `CX_DestructorType`